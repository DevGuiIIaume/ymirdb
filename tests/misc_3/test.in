set a 1
set b a 2
rev b
sort b
uniq b
list entries
bye
