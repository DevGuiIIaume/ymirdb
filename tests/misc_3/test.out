> ok

> ok

> not permitted

> not permitted

> not permitted

> b [a 2]
a [1]

> bye
